Workiva/wGulp
# Do not run coverage on normal test runs

Fixes #55 
## Problem

Karma is instrumenting source files for coverage on every test run. This obfuscates the source code and makes tests hard to debug in browser.
## Solution

Only instrument source files with coverage utils if it is a cover run.

This PR removes the coverage preprocessor and reporter from the default karma.conf.js.
We then dynamically add this config in the `karma` task if a `gulp cover` or `gulp watch:cover` triggered the test run :+1: 
## Testing

The gopher-client repo is a great candidate for testing this as it has 100% coverage
- Ensure `gulp cover` and `gulp watch:cover` produce correct coverage report
- Delete the coverage report, ensure `gulp test` and `gulp watch:test` do not create a coverage report

@trentgrover-wf 
@evanweible-wf 
@shanesizer-wf 

FYI
@patkujawa-wf 
@mikethiesen-wf
